.gitignore

```diff
@@ -9,6 +9,8 @@ scratch/
 bo_eg*.png
 gif/
 
+*.csv
+
```

Comment:
You may remove these lines before doing the merge from my fork, as mentioned earlier